client/components/search/style.scss

```diff
@@ -2,178 +2,106 @@
  * @component Search
  */
 .search {
+	display: flex;
+	flex: 1 1 auto;
 	margin-bottom: 24px;
 	width: 60px;
 	height: 51px;
 	position: relative;
+	align-items: center;
 	// places search above filters
 	z-index: z-index( 'root', '.search' );
+	transition: box-shadow 0.15s;
 
-	@include breakpoint( "<660px" ) {
-		width: 50px;
+	.search__component-icon-div {
```

Comment:
OK will fix. Just one question. In theory one of the benefits of sass is that it allows to write scss files in a way that they represent the same hierarchical structure like HTML does. This should improve readability. Like I thought in code above. Sure that to much nesting in scss will lead to overqualified css but is there a rule like just don't nest or it is a case by case choice to make.